### ISSUE TITLE
404 amirhosseinmohammadi.pdf

### ISSUE THREAD

**Issue by BaseMax** [BlackIQ/BlackIQ#4]
https://github.com/BlackIQ/BlackIQ/raw/main/amirhosseinmohammadi.pdf
https://blackiq.ir/

**Comment by BlackIQ** [comment]
Hi,

Thank you. I'll fix the link. Check https://amirhossein.info instead of https://blackiq.ir.

Tnx.
Amir.

**Comment by BaseMax** [comment]
Hi Amir,

So good.

Best wishes,
M.

**Comment by BaseMax** [comment]
https://blackiq.ir/ is fully down.
https://github.com/BlackIQ/BlackIQ/raw/main/amirhosseinmohammadi.pdf 404

**Comment by BlackIQ** [comment]
Yes, I don't use that domain. The main domain is https://amirhossein.info. And I delete those pdf files. Files are at https://cdn.amirhossein.info/cv. En version is https://cdn.amirhossein.info/cv/amirhossein-mohammadi-cv.en.pdf.

**Comment by BlackIQ** [comment]
Hey there!

How are you? You won't believe it but I added the CV file again 😀. [Commit](https://github.com/BlackIQ/BlackIQ/commit/83fbec3a6ac21f0f3228b09bd8453132af10d4a8)

Now https://github.com/BlackIQ/BlackIQ/raw/main/amirhosseinmohammadi.pdf works!

Also as I mentioned previously, blackiq.ir is down, you can checkout [amirhossein.info](https://www.amirhossein.info) and source code is [here](https://github.com/BlackIQ/amirhossein.info) .

Can't wait for your future issues on any repository! Promise to solve them sooner than 5 years :)

**Comment by BaseMax** [comment]
Super!
